README.md

```diff
@@ -110,6 +110,32 @@ Output :
 | `passedQC`                                   | `true` when QC passed, otherwise `false`                                                        |
 
 
+## Running the pipeline offline
+
+Nextflow can automatically retrieve almost everything necessary to execute a pipeline from the web, including pipeline code, software dependencies, reference genomes, and remote data sources.
+
+However, if your analysis must run on a system without *internet access*, you will need to take a few additional steps to ensure all required components are available locally. First, download everything on an internet-connected system (such as your personal computer) and then transfer the files to the offline system using your preferred method.
+
+To set up an offline environment, you will need three key components: a functioning Nextflow installation, the pipeline assets, and the required reference genomes.
+
+On a computer with an internet connection, to download the pipeline, run:
+
+```bash
+nf-core pipelines download BfArM-MVH/GRZ_QC_Workflow
+```
+
+Add the argument `--container-system singularity` to also fetch the singularity container(s).
```

Comment:
what about docker and conda?

Reply:
conda are being generated automatically during the run using the environments inside the modules so no need to download anything, for docker I dont know, it was only for singularity in the nf-core docs. Maybe it is not possible for docker to run offline

Reply:
Yea this is not possible for docker.

Reply:
Got it :+1:
Nextflow should also pull Docker/Podman containers automatically, but depending on the cluster configuration it might be necessary to provide an offline container registry with all necessary containers.
Can you please include this in the description?

Reply:
I inserted the docs to the main nf-core documentation. I dont want to over explain the things since they have to take care what it is needed to run nextflow pipelines :D

Reply:
OK :grin: